parquet-column/src/main/java/org/apache/parquet/filter/PagedRecordFilter.java

```diff
@@ -52,14 +49,25 @@ private PagedRecordFilter(long startPos, long pageSize) {
     this.endPos   = startPos + pageSize;
   }
 
-  /**
-   * Keeps track of how many times it is called. Only returns matches when the
-   * record number is in the range.
-   */
   @Override
-  public boolean isMatch() {
```

Comment:
@hadrienk I'm afraid that the build fails, because semantic versioning doesn't like the removal of this method. Can you please find a way to keep `isMatch`, while also implement the `UnboundRecordFilter` interface too?